masschallenge/django-accelerator
# [AC-7155]-a Upgrade django-paypal from 0.5 to 1.0

### Changes introduced in: [AC-7155](https://masschallenge.atlassian.net/browse/AC-7155):
- Bumps up django-paypal version

### Merge this PR first

### How to test
- Delete `accelerate_accelerate` image
- Replace `Dockerfile` content in accelerate with the following. This will allow `accelerate` to use `AC-7155` branch on `django-accelerator`
```
FROM 997386117669.dkr.ecr.us-east-1.amazonaws.com/base_accelerate:python3

WORKDIR /srv/www/mc/current

RUN rm -r /django-accelerator && \
    mkdir /django-accelerator && \
    cd /django-accelerator && \
    git init && \
    git pull https://$GITHUB_ACCESS_TOKEN@github.com/masschallenge/django-accelerator.git AC-7155

COPY . /srv/www/mc/current

RUN /bin/bash /srv/www/mc/current/accelerate-provisioner.sh

CMD /srv/www/mc/current/check-mysql-run.sh
```
- Rebuild the image `make build`
- Run `make django-shell`
- In the shell
```
In [1]: import paypal

In [2]: paypal.__version__
Out[2]: '1.0.0'
```
